edc-tests/runtime/iatp/iatp-extensions/src/main/java/org/eclipse/tractusx/edc/iatp/ih/IdentityHubExtension.java

```diff
@@ -15,70 +15,18 @@
 package org.eclipse.tractusx.edc.iatp.ih;
 
 import org.eclipse.edc.identityhub.spi.ScopeToCriterionTransformer;
-import org.eclipse.edc.identityhub.spi.store.CredentialStore;
-import org.eclipse.edc.identityhub.spi.store.model.VerifiableCredentialResource;
-import org.eclipse.edc.identitytrust.TrustedIssuerRegistry;
-import org.eclipse.edc.identitytrust.model.CredentialFormat;
-import org.eclipse.edc.identitytrust.model.CredentialSubject;
-import org.eclipse.edc.identitytrust.model.Issuer;
-import org.eclipse.edc.identitytrust.model.VerifiableCredential;
-import org.eclipse.edc.identitytrust.model.VerifiableCredentialContainer;
 import org.eclipse.edc.runtime.metamodel.annotation.Extension;
-import org.eclipse.edc.runtime.metamodel.annotation.Inject;
 import org.eclipse.edc.runtime.metamodel.annotation.Provider;
 import org.eclipse.edc.spi.system.ServiceExtension;
-import org.eclipse.edc.spi.system.ServiceExtensionContext;
 
-import java.io.IOException;
-import java.time.Instant;
-import java.util.Map;
+import java.util.List;
 
 @Extension("Identity Hub extension for testing")
 public class IdentityHubExtension implements ServiceExtension {
```

Comment:
Is there any reason why the `:runtime:iatp:iatp-extensions` module should _not_ be in the production code base? It would also need sourcedoc and tests then, especially the `*EvaluationFunctions`

Reply:
The evaluation functions are not in production code yet as stated in the comment. They will be implemented/moved in future PRs #962.